### DISCUSSION TITLE
How does this if at all relate to Azure Virtual Network Manager

### DISCUSSION THREAD

**Discussion by cicorias** [Azure/ipam#132] [General]
I've seen Azure Virtual Network Manager in preview. What is the relationship and what gaps does this fill?

https://learn.microsoft.com/en-us/azure/virtual-network-manager/

**Comment by DCMattyG** [comment]
Good morning @cicorias, while this product is not related at all to Azure Virtual Network Manager (AVNM), our team has been working closely with the AVNM team to bring native IPAM support to Azure. AVNM will have IPAM functionality in Preview in the next few months. While this capability within AVNM will be fairly basic to start, it will grow over time and hopefully become as feature rich as our Azure IPAM open-source project.

Hope that answers your question!

**Reply by cicorias** [reply]
It does. Thanks @DCMattyG
